hypercert.json

```
@@ -0,0 +1,113 @@
+            "description": "(Optional) short blurb of the impact work done.",
+            "maxLength": 3000,
+            "maxGraphemes": 300
+          },
+          "description": {
+            "type": "string",
+            "format": "uri",
+            "description": "A description of the impact work done.",
+            "maxLength": 30000,
+            "maxGraphemes": 3000
+          },
+          "image": {
+            "type": "blob",
+            "accept": ["image/*"],
+            "maxSize": 5000000,
+            "description": "Blob reference to hypercert image/visual representation. Maximum size is in bytes"
+          },
+          "workScope": {
+            "type": "string",
+            "description": "Scope of the work performed",
+            "maxLength": 5000,
+            "maxGraphemes": 1000
+          },
+          "workTimeframeFrom": {
+            "type": "string",
+            "format": "datetime",
+            "description": "When the work began"
+          },
+          "workTimeFrameTo": {
+            "type": "string",
+            "format": "datetime",
+            "description": "When the work ended"
+          },
+          "evidence": {
+            "type": "array",
+            "description": "Supporting evidence, documentation, or external data URIs",
+            "items": {
+              "type": "object",
+              "required": ["uri"],
+              "properties": {
+                "uri": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "Evidence resource URI"
+                },
+                "title": {
+                  "type": "string",
+                  "maxLength": 256,
+                  "description": "Optional title to describe the nature of the evidence"
+                },
+                "description": {
+                  "type": "string",
+                  "maxLength": 3000,
+                  "maxGraphemes": 300,
+                  "description": "Optional short description explaining what this evidence demonstrates or proves"
+                }
+              }
+            },
+            "maxLength": 100
+          },
+          "contributors": {
+            "type": "array",
+            "description": "Contributors to this hypercert (names, pseudonyms, or DIDs)",
+            "items": {
+              "type": "string",
+              "maxLength": 2560,
+              "maxGraphemes": 256
+            }
+          },
+          "rights": {
+            "type": "string",
+            "description": "Rights keyword (as defined in app.certified.hypercert.rights)",
+            "maxLength": 10
+          },
+          "location": {
+            "type": "string",
+            "format": "at-uri",
+            "description": "AT URI referencing a location record (app.certified.hypercert.location)"
+          },
+          "createdAt": {
+            "type": "string",
+            "format": "datetime"
```

Think it's worth having a description like the one @holkexyz suggested:

1. to be clear that it's the time the hypercert record was created, rather than anything to do with the timing of the impact work, and also
2. to be explicit that the timestamp is chosen by the client (so could potentially be backdated, for example).

```suggestion
            "format": "datetime",
            "description": "Client-declared timestamp when this hypercert record was originally created"
```